app/src/main/java/de/danoeh/antennapod/dialog/EpisodesApplyActionFragment.java

```diff
@@ -451,4 +457,16 @@ private void close(@PluralsRes int msgId, int numItems) {
         getActivity().getSupportFragmentManager().popBackStack();
     }
 
+    private void closeMore(@PluralsRes int msgId, int countNoMedia, int countHasMedia) {
+        try {
+            ((MainActivity) getActivity()).showSnackbarAbovePlayer(
+                    getResources().getQuantityString(msgId,
+                            countHasMedia,
+                            (countHasMedia + countNoMedia), countHasMedia),
+                    Snackbar.LENGTH_LONG);
+        } catch (Exception e) {
```

Comment:
Why can this throw an exception? Wouldn't those cases indicate a developer error and should make the app crash instead?